lib/diagnostic.rb

```diff
@@ -17,8 +27,9 @@
 # Finally, assign the modified Person to `Answer.dave` below.
 
 ##
-# your answers here
-Answer.dave = nil
+dave = Person.new('Dave', 32, 'Ohio')
+@location = 'Somerville'
```

Comment:
Not quite. Person has a 'setter' method for location (generated by `attr_writer` in Q1) called `location=`.
To set the property value, we can just write `dave.location = "Somerville"`.